service/health/searcher/aws.go

```diff
@@ -2,12 +2,74 @@ package searcher
 
 import (
 	"context"
+
+	"github.com/giantswarm/microerror"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// AWSGreenConditions contains the values for a cluster state which are considered 'green'
+var AWSGreenConditions = map[string]struct{}{
+	"Creating": {}, "Created": {}, "Updated": {}}
+
+// AWSYellowConditions contains the values for a cluster state which are considered 'yellow'
+var AWSYellowConditions = map[string]struct{}{
+	"Scaling": {}, "Draining": {}, "Drained": {}, "Updating": {}}
+
+// AWSRedConditions contains the values for a cluster state which are considered 'red'
+var AWSRedConditions = map[string]struct{}{
+	"Deleting": {}, "Deleted": {}}
+
+const (
```

Comment:
This should be moved inside `key.go` - it will be the same for all providers.

https://github.com/giantswarm/health-service/blob/master/service/health/key/key.go

Reply:
Removed altogether, moved Consts for health 'names' to key